src/app/modals/role/role.component.pug

```diff
@@ -0,0 +1,16 @@
+.modal.fade-in.show
+  .modal-dialog(role="document")
+    .modal-content
+      .modal-header
+        h4.modal-title([innerHtml]="'Complete your Profile' | translate")
```

Comment:
Does this follow our typical capitalization rules? I think in JSTOR we'd use "Complete your profile." Here it seems like it should be at least "Complete Your Profile," if not "Complete your profile."

Reply:
Updated!